### ISSUE TITLE
config.schema doesn't allow SnipeIV 101 (and we can't override into the GUI)

### ISSUE THREAD

**Issue by redking974** [Necrobot-Private/NecroBot#863]
### Expected Behavior:
Let me choose to not snipe several Pokemons

### Actual Behavior:
I got
```
() Fix config.json and restart NecroBot or press a key to ignore and continue...
```

But when I enter into the GUI, nothing happens (we can override into the CLI if we push a key) :
```
[09:26:30] () Enter your command
[09:26:33] () 
```

### Your config.json:
```

    "Snorlax": {
      "SnipeIV": 75,
      "Moves": [],
      "Operator": "or",
      "VerifiedOnly": false,
      "Priority": 5,
      "AutoSnipeCandy": 1,
      "Level": 0,
      "AllowMultiAccountSnipe": false
    },
    "Caterpie": {
      "SnipeIV": 101,
      "Moves": [],
      "Operator": "or",
      "VerifiedOnly": false,
      "Priority": 5,
      "AutoSnipeCandy": 1,
      "Level": 0,
      "AllowMultiAccountSnipe": false
    },
    "Metapod": {
      "SnipeIV": 101,
      "Moves": [],
      "Operator": "or",
      "VerifiedOnly": false,
      "Priority": 5,
      "AutoSnipeCandy": 1,
      "Level": 0,
      "AllowMultiAccountSnipe": false
    },
    "Weedle": {
      "SnipeIV": 101,
      "Moves": [],
      "Operator": "or",
      "VerifiedOnly": false,
      "Priority": 5,
      "AutoSnipeCandy": 1,
      "Level": 0,
      "AllowMultiAccountSnipe": false
    },
    "Kakuna": {
      "SnipeIV": 101,
      "Moves": [],
      "Operator": "or",
      "VerifiedOnly": false,
      "Priority": 5,
      "AutoSnipeCandy": 1,
      "Level": 0,
      "AllowMultiAccountSnipe": false
    },
    "Pidgey": {
      "SnipeIV": 101,
      "Moves": [],
      "Operator": "or",
      "VerifiedOnly": false,
      "Priority": 5,
      "AutoSnipeCandy": 1,
      "Level": 0,
      "AllowMultiAccountSnipe": false
    },
    "Pidgeotto": {
      "SnipeIV": 101,
      "Moves": [],
      "Operator": "or",
      "VerifiedOnly": false,
      "Priority": 5,
      "AutoSnipeCandy": 1,
      "Level": 0,
      "AllowMultiAccountSnipe": false
    },
    "Rattata": {
      "SnipeIV": 101,
      "Moves": [],
      "Operator": "or",
      "VerifiedOnly": false,
      "Priority": 5,
      "AutoSnipeCandy": 1,
      "Level": 0,
      "AllowMultiAccountSnipe": false
    },
    "Spearow": {
      "SnipeIV": 101,
      "Moves": [],
      "Operator": "or",
      "VerifiedOnly": false,
      "Priority": 5,
      "AutoSnipeCandy": 1,
      "Level": 0,
      "AllowMultiAccountSnipe": false
    },
    "Zubat": {
      "SnipeIV": 101,
      "Moves": [],
      "Operator": "or",
      "VerifiedOnly": false,
      "Priority": 5,
      "AutoSnipeCandy": 1,
      "Level": 0,
      "AllowMultiAccountSnipe": false
    },
    "Oddish": {
      "SnipeIV": 101,
      "Moves": [],
      "Operator": "or",
      "VerifiedOnly": false,
      "Priority": 5,
      "AutoSnipeCandy": 1,
      "Level": 0,
      "AllowMultiAccountSnipe": false
    },
    "Paras": {
      "SnipeIV": 101,
      "Moves": [],
      "Operator": "or",
      "VerifiedOnly": false,
      "Priority": 5,
      "AutoSnipeCandy": 1,
      "Level": 0,
      "AllowMultiAccountSnipe": false
    },
    "Venomoth": {
      "SnipeIV": 101,
      "Moves": [],
      "Operator": "or",
      "VerifiedOnly": false,
      "Priority": 5,
      "AutoSnipeCandy": 1,
      "Level": 0,
      "AllowMultiAccountSnipe": false
    },
    "Psyduck": {
      "SnipeIV": 101,
      "Moves": [],
      "Operator": "or",
      "VerifiedOnly": false,
      "Priority": 5,
      "AutoSnipeCandy": 1,
      "Level": 0,
      "AllowMultiAccountSnipe": false
    },
    "Tentacool": {
      "SnipeIV": 101,
      "Moves": [],
      "Operator": "or",
      "VerifiedOnly": false,
      "Priority": 5,
      "AutoSnipeCandy": 1,
      "Level": 0,
      "AllowMultiAccountSnipe": false
    },
    "Doduo": {
      "SnipeIV": 101,
      "Moves": [],
      "Operator": "or",
      "VerifiedOnly": false,
      "Priority": 5,
      "AutoSnipeCandy": 1,
      "Level": 0,
      "AllowMultiAccountSnipe": false
    },
    "Seadra": {
      "SnipeIV": 101,
      "Moves": [],
      "Operator": "or",
      "VerifiedOnly": false,
      "Priority": 5,
      "AutoSnipeCandy": 1,
      "Level": 0,
      "AllowMultiAccountSnipe": false
    },
    "Staryu": {
      "SnipeIV": 101,
      "Moves": [],
      "Operator": "or",
      "VerifiedOnly": false,
      "Priority": 5,
      "AutoSnipeCandy": 1,
      "Level": 0,
      "AllowMultiAccountSnipe": false
    }

```

### (OPTIONAL) Your Log:
```
[09:26:23] () Configuration is up-to-date. Schema version: 13
[09:26:24] () Validating config.json...
[09:26:24] () config.json [Line: 1664, Position: 20]: SnipePokemonFilter.Caterpie.SnipeIV Integer 101 exceeds maximum value of 100.
[09:26:24] () config.json [Line: 1674, Position: 20]: SnipePokemonFilter.Metapod.SnipeIV Integer 101 exceeds maximum value of 100.
[09:26:24] () config.json [Line: 1684, Position: 20]: SnipePokemonFilter.Weedle.SnipeIV Integer 101 exceeds maximum value of 100.
[09:26:24] () config.json [Line: 1694, Position: 20]: SnipePokemonFilter.Kakuna.SnipeIV Integer 101 exceeds maximum value of 100.
[09:26:24] () config.json [Line: 1704, Position: 20]: SnipePokemonFilter.Pidgey.SnipeIV Integer 101 exceeds maximum value of 100.
[09:26:24] () config.json [Line: 1714, Position: 20]: SnipePokemonFilter.Pidgeotto.SnipeIV Integer 101 exceeds maximum value of 100.
[09:26:24] () config.json [Line: 1724, Position: 20]: SnipePokemonFilter.Rattata.SnipeIV Integer 101 exceeds maximum value of 100.
[09:26:24] () config.json [Line: 1734, Position: 20]: SnipePokemonFilter.Spearow.SnipeIV Integer 101 exceeds maximum value of 100.
[09:26:24] () config.json [Line: 1744, Position: 20]: SnipePokemonFilter.Zubat.SnipeIV Integer 101 exceeds maximum value of 100.
[09:26:24] () config.json [Line: 1754, Position: 20]: SnipePokemonFilter.Oddish.SnipeIV Integer 101 exceeds maximum value of 100.
[09:26:24] () config.json [Line: 1764, Position: 20]: SnipePokemonFilter.Paras.SnipeIV Integer 101 exceeds maximum value of 100.
[09:26:24] () config.json [Line: 1774, Position: 20]: SnipePokemonFilter.Venomoth.SnipeIV Integer 101 exceeds maximum value of 100.
[09:26:24] () config.json [Line: 1784, Position: 20]: SnipePokemonFilter.Psyduck.SnipeIV Integer 101 exceeds maximum value of 100.
[09:26:24] () config.json [Line: 1794, Position: 20]: SnipePokemonFilter.Tentacool.SnipeIV Integer 101 exceeds maximum value of 100.
[09:26:24] () config.json [Line: 1804, Position: 20]: SnipePokemonFilter.Doduo.SnipeIV Integer 101 exceeds maximum value of 100.
[09:26:24] () config.json [Line: 1814, Position: 20]: SnipePokemonFilter.Seadra.SnipeIV Integer 101 exceeds maximum value of 100.
[09:26:24] () config.json [Line: 1824, Position: 20]: SnipePokemonFilter.Staryu.SnipeIV Integer 101 exceeds maximum value of 100.
[09:26:24] () Fix config.json and restart NecroBot or press a key to ignore and continue...
[09:26:30] () Enter your command
[09:26:33] () 
```

### Detailed Steps to Reproduce:
Launch Necrobot

Version: 102
Windows Version: 10
What is affected (catch, transfer, favorite, GPX, evolve, recycle, snipe, human walk snipe, pokestops, powerup/levelup, other): launching

**Comment by samuraitruong** [comment]
you can set AutoSnipeMInIV 101

**Comment by redking974** [comment]
Hi @samuraitruong
In fact I would let it to 100 because majority of them are OK for 100 (all of final evolutions of Pokemons that I wouldn't : Ratitac, Golduck, Tentacruel, ...)
I don't want some of them (pre-evolution : Rattata, Aquaduck, Tentacool)
And other that I would more (Snorlax, Lapras)

If I set AutoSnipeMinIV to 101 I have to add a lot more lines because majority are OK for me set to 100.

Hoping my explaination are clear about my issue to set AutoSnipeMinIV to 101

**Comment by samuraitruong** [comment]
ok, not a big deal, I will make that able to enter 101, or ignore .

**Comment by samuraitruong** [comment]
fixed in #882
